engine/class_modules/priest/sc_priest_discipline.cpp

```diff
@@ -424,7 +526,7 @@ void priest_t::create_buffs_discipline()
 
   buffs.harsh_discipline_ready = make_buff( this, "harsh_discipline_ready", find_spell( 373183 ) );
 
-  buffs.borrowed_time = make_buff( this, "borrowed_time", find_spell( 390692 ) )->set_trigger_spell( find_spell( 17 ) );
+  buffs.borrowed_time = make_buff( this, "borrowed_time", find_spell( 390692 ) );
```

Comment:
by removing this you will need to hook it into PW:S manually. Not sure if you've already done that

Reply:
Yes, PW:S triggers it manually.